_targets.R

```diff
@@ -6,39 +6,39 @@ source("3_visualize/src/plot_timeseries.R")
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+site_nums <- c("01427207", "01432160", "01436690", "01466500")
 p1_targets_list <- list(
-  tar_target(
-    site_data,
-    download_nwis_data(),
-  ),
-  tar_target(
-    site_info_csv,
-    nwis_site_info(fileout = "1_fetch/out/site_info.csv", site_data),
-    format = "file"
-  )
+  tar_target(site_data_1, download_nwis_site_data(paste0("1_fetch/tmp/site_data_", site_nums[1], ".csv")), format="file"),
+  tar_target(site_data_2, download_nwis_site_data(paste0("1_fetch/tmp/site_data_", site_nums[2], ".csv")), format="file"),
+  tar_target(site_data_3, download_nwis_site_data(paste0("1_fetch/tmp/site_data_", site_nums[3], ".csv")), format="file"),
+  tar_target(site_data_4, download_nwis_site_data(paste0("1_fetch/tmp/site_data_", site_nums[4], ".csv")), format="file"), 
+  tar_target(site_info_csv_1, nwis_site_info(fileout = "1_fetch/out/site_info_1.csv", site_data_filein = site_data_1), format = "file"), 
+  tar_target(site_info_csv_2, nwis_site_info(fileout = "1_fetch/out/site_info_2.csv", site_data_filein = site_data_2), format = "file"),
+  tar_target(site_info_csv_3, nwis_site_info(fileout = "1_fetch/out/site_info_3.csv", site_data_filein = site_data_3), format = "file"),
+  tar_target(site_info_csv_4, nwis_site_info(fileout = "1_fetch/out/site_info_4.csv", site_data_filein = site_data_4), format = "file")
```

Comment:
I think you appropriately broke the `site_data_[index]` files down so they we could get past the flaky web service. I think that's the one place the pipeline _should_ have gotten more verbose.

For the `site_info_csv_[index]` files, is there a way you can simplify these and all downstream targets? Instead of the 1:1 mapping you currently have (e.g., one `site_info` target for each `site_data` target), I think you could first create a `site_data` target that combines all of the individual `site_data_[index]` targets.

Reply:
I think this allows you to have a single target downstream for each the `site_data_clean`, `site_data_annotated`, and `site_data_styled` groups of targets.

Reply:
Aha, the clue was in nwis_site_info function where it looks at unique stie_no. I was wondering why that was needed. Thanks.

Reply:
@limnoliver about that `empty` file. I thought it was an artifact of something I had done wrong so I deleted them. I brought it back with git checkout from the main branch. But I thought the */out/* folder was supposed to get ignored. When I look at git status they do show up. So, I'm going to commit and push. But, I am a little confused thinking git wasn't tracking the out folders.